ajax/libs/tempusdominus-bootstrap-4/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "license": "MIT",
+  "name": "tempusdominus-bootstrap-4",
+  "author": "Jonathan Peterson",
+  "description": "A date/time picker component designed to work with Bootstrap 4 and Momentjs. For usage, installation and demos see Project Site on GitHub",
+  "homepage": "https://tempusdominus.github.io/bootstrap-4/",
+  "keywords": [
+    "bootstrap",
+    "datepicker",
+    "datetimepicker",
+    "timepicker",
+    "moment"
+  ],
+  "filename": "tempusdominus-bootstrap-4.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tempusdominus/bootstrap-4.git"
+  },
+  "version": "5.0.0-alpha10",
```

Comment:
Not sure if unstable version is okay? Normally cdnjs hopes this to be stable version.